CI/travis.linux.install.sh

```diff
@@ -0,0 +1,5 @@
+#/bin/bash
```

Comment:
:wink: Strangely I recall that we once _had_ a CI/travis.linux.install.sh file but it was _not required_ and take out - just shows that _what goes around, comes around_! 😜